sunpy/coordinates/sun.py

```diff
@@ -94,15 +89,15 @@ def carrington_rotation_time(crot):
     -------
     astropy.time.Time
     """
-    estimate = (_CARRINGTON_ROTATION_PERIOD * (crot - 1)) + _FIRST_CROT_JD
+    estimate = (constants.crot_period.value * (crot - 1)) + constants.first_crot_jd.value
 
     # The above estimate is inaccurate (see comments below in carrington_rotation_number),
     # so put the estimate into carrington_rotation_number to determine a correction amount
     def refine(estimate):
         crot_estimate = carrington_rotation_number(t=Time(estimate, scale='tt', format='jd'))
         dcrot = crot - crot_estimate
         # Correct the estimate using a linear fraction of the Carrington rotation period
-        return estimate + (dcrot * _CARRINGTON_ROTATION_PERIOD)
+        return estimate + (dcrot * constants.crot_period.value)
```

Comment:
```suggestion
        return estimate + (dcrot * mean_synodic_period)
```
Also change the last line of this function to simply
```
    return estimate
```